engine/lib/metastrings.php

```diff
@@ -284,6 +279,14 @@ function _elgg_get_metastring_based_objects($options) {
 		$query = "SELECT {$options['metastring_calculation']}(v.string) as calculation FROM {$db_prefix}$type n_table";
 	}
 
+	foreach ($joins as $i => $join) {
+		if ($join === false) {
```

Comment:
i don't know - i thought it was probably something thrown in there to fix something specific from a while ago. i don't know what exactly it would fix.